fzf-marks.plugin.bash

```diff
@@ -22,12 +22,25 @@
 
 command -v fzf >/dev/null 2>&1 || return
 
+PROGRAM_NAME="${PROGRAM_NAME:-fzf-marks}"
+
+CONFIG_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/$PROGRAM_NAME" #currently unused, but in the future can be used to load (and store?) an .env file from to more easily switch between multiple configuration
+DATA_DIR="${XDG_DATA_HOME:-$HOME/.local/share}/$PROGRAM_NAME"
+LEGACY_FZF_MARKS_FILE="${HOME}/.fzf-marks"
+
 if [[ -z ${FZF_MARKS_FILE-} ]] ; then
-    FZF_MARKS_FILE=${HOME}/.fzf-marks
+    FZF_MARKS_FILE="${DATA_DIR}/default_bookmarks.fzf-marks"
 fi
 
-if [[ ! -f ${FZF_MARKS_FILE} ]]; then
+if [[ ! -f "${FZF_MARKS_FILE}" ]]; then
```

Comment:
These quotes (`"`) are not needed in the conditional construct `[[ ... ]]`. In the conditional construct, arguments are not subject to word splitting and pathname expansions.

Similarly, the right-hand side of the variable assignments is neither subject to word splitting and pathname expansions, so the quotes in `FZF_MARKS_FILE="${DATA_DIR}/default_bookmarks.fzf-marks"` are also unneeded.

Reply:
I guess I'm just used to using double quotes where possible to prevent word splitting. :)
Is there a downside to using quotes inside the conditional construct?

Reply:
Actually, I think there is no downside to quote the parameter expansion, so yeah, this is a matter of preference. One of the reasons that the keyword `[[` is introduced as a variant of the command `[` is to resolve quotation problems in `[`, so I just felt it natural to quote the arguments only when necessary.

Reply:
@akinomyoga makes a good point. At some point in the future, I'll go through the codes and make sure we are consistent everywhere. Same with $FZF_MARKS_FILE vs ${FZF_MARKS_FILE}. Since the quotes, and in the latter case the curly brackets, are unnecessary, we could do away with them, but for this pull request this does not matter much.